frontend/src/Components/OperationalInsightsCharts.tsx

```diff
@@ -35,6 +34,9 @@ const OperationalInsights = () => {
 
     const metrics = data?.data;
 
+    const totalUsers =
```

Comment:
I'm getting a typescript error here because metrics is either `LoginMetrics | undefined`. Is there a way to be clear this isn't undefined?